PED/SetPedMinGroundTimeForStungun.md

```diff
@@ -8,11 +8,84 @@ ns: PED
 void SET_PED_MIN_GROUND_TIME_FOR_STUNGUN(Ped ped, int ms);
 ```
 
+Overwrites the minimum time the ped will stay on the ground for after being stunned. Setting this while the ped is stunned will not alter the duration of the current stun (but will still effect future stunns).
+
+Unlike what this native's name suggests, it also effects the time for other "weapons", such as WEAPON_ELECTRIC_FENCE.
+
+Passing a negative value (usually -1) into the second parameter (ms) will reset the ground time back to default (ground time will then depend on the weapon that inflicts the stun).
+
+## Parameters
+* **ped**: The ped to set the min ground time for
+* **ms**: The minimum time in milliseconds
```

Comment:
Reword with "The minimum stun effect duration in milliseconds"

Reply:
I implemented Avarian's suggestion, but this nevertheless would have been an improvement over the original.